community-concepts/productaanvraag/productaanvraag-denhaag.json

```
@@ -0,0 +1,89 @@
+{
+    "title": "Productaanvraag tbv terugbelnotitie Den Haag",
+    "default": {},
+    "examples": [
+        {
+            "data": {
+                "naam": "Jan Jansen",
+                "omschrijving": "Ik heb een vraag over mijn paspoort",
+                "telefoonnummer": "0612345678"
+            },
+            "type": "terugbelnotitie",
+            "bsn": "111222333",
+            "pdf_url": "https://documenten.nl/api/v1/enkelvoudiginformatieobjecten/230bab4a-4b51-40c6-91b2-f2022008a7f8",
+            "attachments": [],
+            "submission_id": "a43e84ac-e08b-4d5f-8d5c-5874c6dddf56"
+        }
+    ],
+    "required": [
+        "submission_id",
+        "type",
+        "data"
+    ],
+    "properties": {
+        "data": {
+            "$id": "#/properties/data",
+            "type": "object",
+            "title": "Object met de ingezonden formulierdata",
+            "default": {},
+            "examples": [
+                {
+                    "field1": "value1"
+                }
+            ]
+        },
+        "type": {
+            "$id": "#/properties/type",
+            "type": "string",
+            "title": "Type productaanvraag",
+            "default": "",
+            "examples": [
+                "terugbelnotitie"
+            ]
+        },
+        "bsn": {
+            "$id": "#/properties/bsn",
+            "type": "string",
+            "title": "Burgerservicenummer",
+            "default": "",
+            "examples": [
+                "111222333"
+            ]
+        },
+        "pdf_url": {
+            "$id": "#/properties/pdf_url",
+            "type": "string",
+            "title": "URL van de pdf van bevestiging in Open Forms",
+            "format": "uri",
+            "default": "",
+            "examples": [
+                "https://openforms.nl/media/submission.pdf"
```

```suggestion
                "https://example.com/media/submission.pdf"
```

I don't like semi-real URLs. It's either a public URL or example.com